source/ada/lsp-ada_contexts.adb

```diff
@@ -467,14 +467,43 @@ package body LSP.Ada_Contexts is
    ---------------------------
 
    function Find_All_Env_Elements
-     (Self     : Context;
-      Name     : Libadalang.Analysis.Name;
-      Seq      : Boolean := True;
-      Seq_From : Libadalang.Analysis.Ada_Node'Class :=
-        Libadalang.Analysis.No_Ada_Node)
-     return Libadalang.Analysis.Ada_Node_Array is
```

Comment:
Let's use sets instead

Reply:
This is now using a set for filtering duplicate and a vector to preserve the order.